scripts/.latest-release

```diff
@@ -0,0 +1 @@
+v0.0.9
```

Comment:
Maybe instead using a `build` dir within the root similar to https://github.com/kubernetes/kompose/blob/master/build/VERSION which will contain a RELEASE.md on instructions how to release?

@kadel